clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -388,6 +390,9 @@ public void close(long duration, TimeUnit unit) {
         try {
             // Wait for the thread to be joined.
             thread.join();
+
+            AppInfoParser.unregisterAppInfo(JMX_PREFIX, clientId);
+            
             log.debug("{}: closed.", clientId);
```

Comment:
Yes, I know, but the other messages in the `close` statement follow this pattern. So, it seemed more important to be consistent with the other log messages in this class than other classes.